app/main.py

```diff
@@ -1,34 +1,84 @@
+from typing import Tuple, List
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(
+            self,
+            row: int,
+            column: int,
+            is_alive: bool = True
+    ) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: Tuple[int, int],
+            end: Tuple[int, int],
+            is_drowned: bool = False
+    ) -> None:
+        self.decks: List[Deck] = []
+        self.is_drowned = is_drowned
+        horizontal = start[0] == end[0]
+        row_range = range(start[0], end[0] + 1) \
+            if not horizontal else [start[0]]
```

Comment:
Avoid using line breakers in your code, make a multi-line if condition or place it inside `()`